src/server/api/routes/qr.router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+const qrController = require('../controllers/qr.controller');
+
+/**
+ * @swagger
+ * /qr/{GAME}:
+ *  get:
+ *    summary: Get the qr code accocieted with game instance.
+ *    description:
```

Comment:
Small spelling mistake. **accocieted => associated**

Reply:
Done, thank you

Reply:
Done, thanks